lib/rules/color-no-out-gamut-range/__tests__/index.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const { messages, ruleName } = require('../index.js');
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: '@media (color-gamut: p3) {\n' + 'a {\n' + 'color: lch(48% 82 283);\n' + '}\n' + '}',
```

Comment:
[suggest] Could you please use `stripIndent` instead of `\n` for readability?

For example:

https://github.com/stylelint/stylelint/blob/dab8df2b3bf58e545f683a0f49d821abc089cde3/lib/rules/color-function-notation/__tests__/index.js#L44-L48